app/assets/javascripts/float-kit/addon/components/d-default-toast.gjs

```diff
@@ -12,6 +12,9 @@ const DDefaultToast = <template>
     }}
     ...attributes
   >
+    {{#if @data.progressBar}}
```

Comment:
It's annoying to have this here but I get why you did this

Reply:
I could append from the parent component via JS I guess, I don't really see it as an issue but it would work as an alternative since we are simply adding an element.